db/migrate/20180926213507_create_rents.rb

```diff
@@ -0,0 +1,12 @@
+class CreateRents < ActiveRecord::Migration[5.1]
+  def change
+    create_table :rents do |t|
+      t.references :user, foreign_key: true
```

Comment:
We don't want any of these columns to be null, let's add a `null: false` restriction to each one and a validation on the model to be sure that every attribute is present when an instance of the model is created.